tests/models/whisper/test_processor_whisper.py

```diff
@@ -146,3 +146,11 @@ def test_get_decoder_prompt_ids(self):
 
         expected_ids = [TRANSCRIBE, NOTIMESTAMPS]
         self.assertListEqual([ids[-1] for ids in forced_decoder_ids], expected_ids)
+
+    def test_get_prompt_ids(self):
```

Comment:
Could you also add some tests for edge cases?

For example: `processor.get_prompt_ids("")` or `processor.get_prompt_ids("<|startofprev|> Mr. <|startofprev|> Quilter")`

Reply:
The second will definitely confuse the model and decoding if they were passed to the current `get_prompt_ids` as is, would you prefer we strip the prompt start token or raise an error that it was included? I'll push up a change that strips it and logs a warning for now

Reply:
I don't really know what would be the best approach here, was just trying to think of things that might go wrong. ;-) 

Perhaps raising an error on unexpected input is the best choice, but only if it doesn't add a lot of complexity to the code.

Reply:
Looks like they have their tiktoken package handle it and it raises an error if any special token is included, so will look to do the same